client/public/locales/en/translation.json

```diff
@@ -510,6 +510,10 @@
       "save": "Failed to save {{type}}."
     }
   },
+  "tooltip": {
+    "priority": "Tasks priority(Low-High). Impacts tasks scheduling policy.",
```

Comment:
Hi, I am not sure about this description regarding the priority value.The values can range from zero to any positive whole number.
`Low-High` doesn't say much to me. wdyt?

Reply:
Suggestion: Please remove `(Low-High)`. The user can check the release notes documentation that says: `Priority: Priority of the task. The value is from zero to any positive integer. The higher the value in this column, the higher the priority of the task.`

Or use something like: `Tasks priority, 0 to any positive integer (Low-High). Impacts tasks scheduling policy.` as the tooltip

Reply:
Change made in #2028